extensions/ql-vscode/src/model-editor/modeling-store.ts

```diff
@@ -347,6 +347,7 @@ export class ModelingStore extends DisposableObject {
     }
 
     return {
+      databaseItem: dbState.databaseItem,
```

Comment:
I don't think we're using this but I also don't mind us adding it.

Reply:
Thanks for noticing this, I was intending to use it. Right now, when the webview is loaded it could result in an invalid state because the webview has a different state than the view provider.